src/main/resources/com/cloudbees/hudson/plugins/folder/buildable/DisableAction/index.jelly

```diff
@@ -0,0 +1,22 @@
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
+         xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+  <l:layout title="${%Disable.title(it.item.getDisplayName())}">
+    <st:include page="sidepanel.jelly" it="${it.item}"/>
+    <l:main-panel>
+      <h1>
+        ${%Disable.title(it.item.getDisplayName())}
+      </h1>
+
+      <form action="disable" method="POST">
+      ${%confirmationMessage}
+        <ul>
+          <j:forEach var="it" items="${it.getJobs()}">
```

Comment:
This probably shows fewer jobs than will get disabled (nested folders?). In general, this should probably not use a different source for the list of items than the actual action.

Plus, false positives in case of the already mentioned CCE could be caught and listed separately.